src/pybind/mgr/mgr_module.py

```diff
@@ -648,6 +738,12 @@ def set_health_checks(self, checks):
         """
         self._ceph_set_health_checks(checks)
 
+    def _handle_command(self, inbuf, cmd):
+        for cli_cmd in CLICommand.COMMAND_LIST:
```

Comment:
For the sake of simplicity, what do you think of making this the default implementation of `handle_command` without introducing `_handle_command`? The dashboard module is the only one that would need to overwrite it.

Reply:
By setting the default implementation for `handle_command`, we're basically saying that no one should overwrite `handle_command` anymore. This would in my opinion be the proper way of making sure, every module is going to use `CLICommand` eventually.

Reply:
Since the `@CLICommand` decorator still does not support the `inbuf` parameter of commands, I don't want to change the `handle_command` contract right away. Let's make this "default implementation" change in a future PR where we also introduce the support for the inbuf parameter.

Reply:
And why not simply offering the Ceph-Mgr module developers with this new helper class `CLICommand` and a new ready-to-use method `handle_command()`, so they only need to replace their `handle_command` implementations with this recipe?:
```python
    def handle_command(self, inbuf, cmd):
        return CLICommand.handle_command(inbuf, cmd)
```

There are several ways not to break existing modules, but this way we are making everything implicitly: we are dropping one *abstract* method from the API to replace it with a completely different *final* method, while trying to keep users completely out of this. I'm not a Python bigot, but one of few lines I fully agree with in the Zen of Python is the one about "_explicit is better than implicit_".